Tests/Tests/STPSourceTest.m

```diff
@@ -111,12 +33,11 @@ - (void)testDecodingSource_ideal {
     XCTAssertEqual(source.status, STPSourceStatusPending);
     XCTAssertEqual(source.type, STPSourceTypeIDEAL);
     XCTAssertEqual(source.usage, STPSourceUsageSingleUse);
-    XCTAssertEqualObjects(source.details, response[@"ideal"]);
+    XCTAssertEqualObjects(source.details[@"bank"], @"ing");
```

Comment:
I'll also add some fixtures tests to make sure `allResponseFields` matches the initial dictionary.